app/views/users/_form.html.erb

```diff
@@ -97,11 +97,11 @@
           <%= f.label :agreement, 'Foster Agreement', :class => 'control-label' %>
           <div class="controls">
           <%= f.fields_for :agreement do |af| %>
-            <% if af.object.new_record? %>
+            <% if af.object.persisted? %>
               <%= af.file_field :attachment %>
```

Comment:
you'll have 2 input fields if you leave this no?  you basically want the conditional to show the overwrite? if one exists already right?  trying to remove having to have the line `<%= af.file_field :attachment %>` twice